pages/project/read/views/sections/downloads.jsx

```diff
@@ -138,7 +140,8 @@ export default function Downloads() {
 
           {
             // latest version might be application, amendment or granted
-            latestVersion && <DownloadSection project={project} version={latestVersion} />
+            latestVersion && !(grantedVersion.id !== latestVersion.id && viewingAtAAEstablishment) &&
```

Comment:
What's the reasoning behind the extra `grantedVersion.id !== latestVersion.id` condition?

Reply:
If you're viewing at the primary establishment, then you should see download links to the latest version, and if the latest version is not granted, then we also need to show a section for the granted version.

If you're viewing at an AA establishment, then we should only show the granted version, regardless of amendments in progress.